clang/test/Driver/arm-fpu-selection.s

```diff
@@ -0,0 +1,32 @@
+// REQUIRES: arm-registered-target
+// Ensures that when targeting an ARM target with an Asm file, clang
+// collects the features from the FPU. This is critical in the
+// activation of NEON for supported targets. The Cortex-R52 will be
+// used and tested for VFP and NEON Support
+
+// RUN: %clang -target arm-none-eabi -mcpu=cortex-r52 -c %s -o /dev/null | count 0
```

Comment:
The purpose of this line is what exactly?

If we have errors the RUN line will fail because clang returns non-zero, so perhaps a check for warnings? Should you be piping stderr as well if that's the case?